src/main/java/org/folio/consortia/repository/SharingRoleRepository.java

```diff
@@ -10,9 +11,19 @@
 
 public interface SharingRoleRepository extends JpaRepository<SharingRoleEntity, UUID> {
 
+  @Query("SELECT sr FROM SharingRoleEntity sr WHERE sr.roleId = ?1")
+  List<SharingRoleEntity> findByRoleId(UUID roleId);
+
+  @Query("SELECT sr FROM SharingRoleEntity sr WHERE sr.roleId = ?1 and sr.tenantId = ?2")
```

Comment:
these 2 queries can be deleted, since Spring Data will construct sql queries based on method name convention
@Query("SELECT sr FROM SharingRoleEntity sr WHERE sr.roleId = ?1")
@Query("SELECT sr FROM SharingRoleEntity sr WHERE sr.roleId = ?1 and sr.tenantId = ?2")